apps/experiments/tasks.py

```diff
@@ -23,6 +27,44 @@ def get_response_for_webchat_task(self, experiment_session_id: int, message_text
     return message_handler.new_user_message(message)
 
 
+@shared_task(bind=True, base=TaskbadgerTask)
+def store_rag_embedding(self, experiment) -> None:
```

Comment:
Rather than taking the experiment object this should take just the ID and fetch the experiment from the DB:

```suggestion
def store_rag_embedding(self, experiment_id: int) -> None:
    experiment = Experiment.objects.get(id=experiment_id)
```